code/obj/item/sec_tape.dm

```diff
@@ -0,0 +1,144 @@
+			var/obj/sec_tape/T1 = new /obj/sec_tape(T, src)
+			T1.set_dir(EAST)
+			var/obj/sec_tape/T2 = new /obj/sec_tape(T, src)
+			T2.set_dir(WEST)
+			var/obj/sec_tape/T3 = new /obj/sec_tape(T, src)
+			T3.set_dir(NORTH)
+			var/obj/sec_tape/T4 = new /obj/sec_tape(T, src)
+			T4.set_dir(SOUTH)
+			//No need to set it up to the south, it already is by default
+			user.death(FALSE)
+			qdel(src)
+		else
+			//Else we'll choke ourselves out
+			user.visible_message("<span class='alert'><b>[user] wraps some tape around \his neck and tightens it.</b></span>")
+			user.take_oxygen_deprivation(160)
+			SPAWN(50 SECONDS)
+				if (user && !isdead(user))
+					user.suiciding = 0
+		return 1
+
+	update_icon()
+		inventory_counter?.update_number(amount)
+
+/obj/item/sec_tape/vended //Tape in secvends has a pre-set amount of uses
+	New(loc)
+		..(loc)
+		amount = 20
+		src.UpdateIcon()
+
+/obj/item/sec_tape/get_desc()
+	return "There's [amount] length[s_es(amount)] left."
+
+/obj/item/sec_tape/attackby(obj/item/W, mob/user)
+	if (issnippingtool(W) && src.amount > 1) //Cut some of it off to share
+		var/obj/item/sec_tape/split_tape = split_stack(1)
+		user.put_in_hand_or_drop(split_tape)
+		tooltip_rebuild = 1
+		boutput(user, "<span class='notice'>You cut a piece off the [base_name].</span>")
+		src.UpdateIcon()
+		return
+
+	else if (istype(W, /obj/item/sec_tape))
+		var/obj/item/sec_tape/C = W
+
+		if (C.amount == src.max_stack)
+			boutput(user, "<span class='notice'>The coil is too long, you cannot add any more tape to it.</span>")
+			return
+
+		if ((C.amount + src.amount <= src.max_stack))
+			stack_item(W)
+			boutput(user, "<span class='notice'>You join the tape ends together.</span>")
+			C.tooltip_rebuild = 1
+			C.UpdateIcon()
+			return
+
+		else
+			boutput(user, "<span class='notice'>You transfer [src.max_stack - src.amount] length\s of tape from one roll to the other.</span>")
+			src.amount -= (src.max_stack-C.amount)
+			src.UpdateIcon()
+			tooltip_rebuild = 1
+			C.amount = src.max_stack
+			C.UpdateIcon()
+			C.tooltip_rebuild = 1
+			return
+
+/obj/item/sec_tape/MouseDrop_T(atom/movable/O as obj, mob/user as mob)
+	..(O, user)
+
+/obj/item/sec_tape/afterattack(turf/F, mob/user)
+	if (!isturf(user.loc))
+		return
+
+	if (!istype(F,/turf/simulated/floor))
+		return
+
+	if (BOUNDS_DIST(F, user) > 0)
+		boutput(user, "<span class='notice'>You can't setup a cordon at a place that far away.</span>")
+		return
+
+	var/dirn = user.dir
+	for (var/obj/sec_tape/T in F)
+		if (T.dir == dirn)
+			boutput(user, "<span class='notice'>There is already a cordon setup there!</span>")
+			return
+
+	var/obj/sec_tape/ST = new /obj/sec_tape(F, src)
+	if (dirn == SOUTH)
+		ST.set_dir(SOUTH)
+	else if (dirn == EAST)
+		ST.set_dir(EAST)
+	else if (dirn == WEST)
+		ST.set_dir(WEST)
+	else
+		ST.set_dir(NORTH)
+	boutput(user, "<span class='notice'>You [pick("hastily", "quickly", "haphazardly")] setup a security cordon.</span>")
```

Comment:
```suggestion
	boutput(user, "<span class='notice'>You [pick("hastily", "quickly", "haphazardly")] setup a security cordon.</span>", group="sectape_setup")
```
This makes it so the chat messages collapse and get the [x4] counter next to them even if their contents don't match exactly.